### DISCUSSION TITLE
Problem with new module from tuya

### DISCUSSION THREAD

**Discussion by Liionboy** [ct-Open-Source/tuya-convert#1037] [Q&A]
Hello,

Anyone figured out how to put tasmota on a Tuya CBU Module? I have some ZMAI 90 and all of them come with a new CBU module.

https://adinextcloud.go.ro/s/BocBJr5PNiso444  picture with CBU Module

Thanks

**Comment by Wolle62** [comment]
I have the same device and tried to exchange the CBU module for an ESP12. Unfortunately I don't get any values ​​over the serial interface with Tasmota. Has anyone done it yet?

**Comment by koffing** [comment]
Do you guys know what MCU is used for energy metering? mine is erased on the new model with CBU module. They used to have V9821 on older model
